db_stress_tool/myrocks_style_txns_stress.cc

```diff
@@ -0,0 +1,1322 @@
+}
+
+Status MyRocksStyleTxnsStressTest::TestPrefixScan(
+    ThreadState* thread, const ReadOptions& read_opts,
+    const std::vector<int>& rand_column_families,
+    const std::vector<int64_t>& rand_keys) {
+  (void)thread;
+  (void)read_opts;
+  (void)rand_column_families;
+  (void)rand_keys;
+  return Status::OK();
+}
+
+// Given a key K, this creates an iterator which scans to K and then
+// does a random sequence of Next/Prev operations.
+Status MyRocksStyleTxnsStressTest::TestIterate(
+    ThreadState* thread, const ReadOptions& read_opts,
+    const std::vector<int>& /*rand_column_families*/,
+    const std::vector<int64_t>& /*rand_keys*/) {
+  uint32_t c = thread->rand.Next() % kInitNumC;
+  return RangeScanTxn(thread, read_opts, c);
+}
+
+// Not intended for use.
+Status MyRocksStyleTxnsStressTest::TestPut(
+    ThreadState* /*thread*/, WriteOptions& /*write_opts*/,
+    const ReadOptions& /*read_opts*/, const std::vector<int>& /*cf_ids*/,
+    const std::vector<int64_t>& /*keys*/, char (&value)[100],
+    std::unique_ptr<MutexLock>& /*lock*/) {
+  (void)value;
+  return Status::NotSupported();
+}
+
+// Not intended for use.
+Status MyRocksStyleTxnsStressTest::TestDelete(
+    ThreadState* /*thread*/, WriteOptions& /*write_opts*/,
+    const std::vector<int>& /*rand_column_families*/,
+    const std::vector<int64_t>& /*rand_keys*/,
+    std::unique_ptr<MutexLock>& /*lock*/) {
+  return Status::NotSupported();
+}
+
+// Not intended for use.
+Status MyRocksStyleTxnsStressTest::TestDeleteRange(
+    ThreadState* /*thread*/, WriteOptions& /*write_opts*/,
+    const std::vector<int>& /*rand_column_families*/,
+    const std::vector<int64_t>& /*rand_keys*/,
+    std::unique_ptr<MutexLock>& /*lock*/) {
+  return Status::NotSupported();
+}
+
+void MyRocksStyleTxnsStressTest::TestIngestExternalFile(
+    ThreadState* thread, const std::vector<int>& rand_column_families,
+    const std::vector<int64_t>& /*rand_keys*/,
+    std::unique_ptr<MutexLock>& /*lock*/) {
+  // TODO (yanqin)
+  (void)thread;
+  (void)rand_column_families;
+}
+
+void MyRocksStyleTxnsStressTest::TestCompactRange(
+    ThreadState* thread, int64_t /*rand_key*/, const Slice& /*start_key*/,
+    ColumnFamilyHandle* column_family) {
+  // TODO (yanqin).
+  // May use GetRangeHash() for validation before and after DB::CompactRange()
+  // completes.
+  (void)thread;
+  (void)column_family;
+}
+
+Status MyRocksStyleTxnsStressTest::TestBackupRestore(
+    ThreadState* thread, const std::vector<int>& rand_column_families,
+    const std::vector<int64_t>& /*rand_keys*/) {
+  // TODO (yanqin)
+  (void)thread;
+  (void)rand_column_families;
+  return Status::OK();
+}
+
+Status MyRocksStyleTxnsStressTest::TestCheckpoint(
+    ThreadState* thread, const std::vector<int>& rand_column_families,
+    const std::vector<int64_t>& /*rand_keys*/) {
+  // TODO (yanqin)
+  (void)thread;
+  (void)rand_column_families;
+  return Status::OK();
+}
```

Comment:
It's interesting many of these need to be repeated across different `StressTest`s. I will think more about this later...